entgql/annotation.go

```diff
@@ -51,6 +51,10 @@ type (
 		QueryField *FieldConfig `json:"QueryField,omitempty"`
 		// MutationInputs defines the input types for the mutation.
 		MutationInputs []MutationConfig `json:"MutationInputs,omitempty"`
+		// AllowedOps limits the predicate operations that will be generated for this field.
+		// Ops will be generated using the field type. Then, if this is defined, they will
+		// be filtered by the allowed ops.
+		AllowedOps []gen.Op
```

Comment:
Let’s redefine the ops as bitflag and use it here. This fits better with the current API, and also avoid using `entc/gen` which most users are not familiar with.

Reply:
Hi! I added this as a new `entgql.Ops` type and did not modify entc/gen. Instead, entgql.Ops performs a mapping between itself and the entc/gen.Op type. Please let me know if you wanted entc/gen.Op to be removed or switched to bitwise too with this PR.